File: Basics/Conditional-Statements-Advanced/Lab/number_in_range.py
number = int(input())

is_zero = False
is_in_range = False

if -100 <= number <= 100:
    is_in_range = True
    if number == 0:
        is_zero = True

if is_zero:
    print("No")
elif is_in_range:
    print(f"Yes")
else:
    print(f"No")

File: JS-Basics/0-3-Conditional-Steatements-Advanced/Lab/number-in-range.js
function isNumInRange(input) {
    let n = Number(input[0]);
    let theRange = [];
    for (let i = -100; i < 101; i++) {
        theRange.push(i);
    }

    if (theRange.includes(n) && n != 0) {
        console.log("Yes");
    } else {
        console.log("No");
    }
}



isNumInRange(["-25"])
isNumInRange(["0"])
isNumInRange(["25"])